### DISCUSSION TITLE
reclassify land cover maps into mHM LC-classes

### DISCUSSION THREAD

**Discussion by calvarezgarreton** [mhm-ufz/mHM#31] [Q&A] [answered]
Dear mHM team, 

I am mHM running in Chile with global morph data sets provided by Olda, and local forcing datasets (Q, Pr, T, PET). With quite promising results :)!

I am now replacing some of the global morph datasets with local datasets available for Chile to further improve model realism & performance.

1) For the case of land cover, I am reclassifying 30-m landsat based land cover map from Zhao et al., 2016 [https://www.sciencedirect.com/science/article/abs/pii/S0034425716302188?via%3Dihub] into the 3 mHM land cover classes at a 0.001953125 pixel res 

2) For LAI, we are building monthly 1-km dataset based on different data sources (land cover combined with satellite lai products). We will rescale this data into 0.001953125 pixel res. I'm running the model with the PET-LAI correction option to better represent the vegetation dynamics within similar land cover classes.  

I would like to ask for your advice on how to classify **water bodies, snow & ice** into mHM classes (1=forest, 2=impervious, 3=pervious). Should I just set them as 3, and then rely on the PET-LAI correction option to constrain the ET estimation?

Any comment/tip would be much appreciated. 
Thanks!
Camila

**Comment by rohinikumar82** [comment, marked as answer]
Dear @calvarezgarreton,

Nice to hear about mHM application (with promising results) in Chile catchments... 

Yes at the moment, you can assign "water bodies, snow & ice" into mHM classes 3 (=pervious). 

Best,
Rohini

**Reply by rohinikumar82** [reply]
Dear @calvarezgarreton,
Can you please close this issue using "answered" option - of course once you are satisfied with the answer.
thanks.
Best,
Rohini
